.github/workflows/build-docs.yml

```diff
@@ -3,31 +3,7 @@ name: Docs
 on: [push, workflow_dispatch]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-python@v5
-        with:
-          python-version: 3.x
-
-      - name: Install docs dependencies
-        run: pip install -r docs/requirements.txt -e .
```

Comment:
The docs here require the cpp-linter package installed because the CLI doc is generated from cpp-linter/cli.py module. Thus the `-e .` here.